### ISSUE TITLE
Thresholding score to get predictions.

### ISSUE THREAD

**Issue by crishna0401** [xuhongzuo/scale-learning#1]
Hi, how do you threshold the score function to get the final labels? Please provide a sample script replicating the result presented in paper on any dataset.

**Comment by xuhongzuo** [comment]
We use AUC-ROC and AUC-PR as evaluation metrics, they can directly measure anomaly scores. Threshold is not required to replicate the results presented in the paper.  


---- Replied Message ----
| From | Krishna ***@***.***> |
| Date | 9/4/2023 12:37 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [xuhongzuo/scale-learning] Thresholding score to get predictions. (Issue #1) |

Hi, how do you threshold the score function to get the final labels? Please provide a sample script replicating the result presented in paper on any dataset.

**Comment by crishna0401** [comment]
OK, understood. For AUC-PR metric what function you are using? I used average_precision_score  from sklearn, I am getting different results.

**Comment by xuhongzuo** [comment]
you should use `main.py`. This script contains data importing, pre-processing, training, inference, and evaluation.

**Comment by crishna0401** [comment]
Hey thanks for quick reply. I just looked at the API type code. I didn't use the main.py.
